source/JSONFaceSettings.mc

```diff
@@ -0,0 +1,82 @@
+import Toybox.WatchUi;
+
+    var lineTranslations = new [10];
```

Comment:
Is it possible to get these values (10 and 8) dynamically from the number of keys the two enums have? I hope so.

Reply:
For line translations I actually think that we might add translation string with just "Line" and then for the watch UI we can dynamically add the number so instead of array[10] we will have just a single variable.
Unfortunately we cannot do that for settings files and each line translation would have to stay.

Additional strings.xml entry:
`<string id="line">Line</string>`

As for valueTranslations size we might add a constant to the FeatureEnum class and update it when new feature is added. 

I cannot find any other way to find those values dynamicaly.

I have made a commit with some changes regarding this. 
I hope that MAX_LINES_COUNT won't need to be changed as it is linked to how many properties there are. And those have to be staticaly defined to work in IQ store.

FeatureEnum contains now FEATURES_COUNT, this way we don't have to use those magic numbers in other places.